packages/contracts-bedrock/deploy-config/boba-devnet.json

```diff
@@ -0,0 +1,57 @@
+{
```

Comment:
Nit: but the other files in this directory seem to use the l1-chain name in them, so maybe this should be `boba-sepolia-devnet.json` or similar? Obviously not a big deal though, might be more of a pain to change at this point than it's worth.

Reply:
Ok! Changed.